aztec-up/bin/aztec

```diff
@@ -1,21 +1,39 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-# Call cli image if used with `aztec cli ...args`
-if [ -n "${1-}" ] && [ "$1" != "--help" ]; then
-  if [ "$1" == "cli" ]; then
-    shift
-    SKIP_PORT_ASSIGNMENT=1 $(dirname $0)/.aztec-run aztecprotocol/cli "$@"
-  elif [ "$1" == "sandbox" ]; then
-    $(dirname $0)/aztec-sandbox
-  else
-    $(dirname $0)/.aztec-run aztecprotocol/aztec "$@"
-  fi
-else
-  # TODO - display help message
-  echo
-  echo "Using 'aztec' CLI:"
-  echo "  aztec start <args>  - Start aztec infrastructure components. See 'aztec start --help' for detailed command info."
-  echo "  aztec sandbox       - Run a local sandbox network (same as aztec-sandbox)."
-  echo "  aztec cli <args>    - Run the aztec client CLI. See 'aztec cli --help' for detailed command info."
+function get_compose {
+  # Favour 'docker compose', falling back on docker-compose.
+  CMD="docker compose"
+  $CMD &>/dev/null || CMD="docker-compose"
+  $CMD $@
+}
+
+CALLED_FROM=$PWD
+
+if [ "${1:-}" == "test" ]; then
+  # Change working dir, so relative volume mounts are in the right place.
+  cd $(dirname $0)/..
+  # Compose file to use
+  FILE_ARG="-f $HOME/.aztec/docker-compose.test.yml"
```

Comment:
I'd change this to a relative path since we're in the folder containing the script. This would allow devs to keep the aztec files wherever they want on disk :D

Reply:
Alternatively to keep the absolute path for traps could do something like `FILE_ARG="-f $(realpath ./docker-compose.test.yml)"` could work

Reply:
mmm, I think we force users to install the sandbox in their homes by design, and don't really want them to be able to use it elsewhere. Some have actually complained that we mount their home directory in our container, so I would probably tackle that in a separate PR